src/workerd/api/tests/js-rpc-test.js

```diff
@@ -483,3 +483,17 @@ export let crossContextSharingDoesntWork = {
     await assert.rejects(() => env.MyService.tryUseGlobalRpcPromisePipeline(), expectedError);
   },
 }
+
+// Test that exceptions thrown from async native functions have a proper stack trace. (This is
+// not specific to RPC but RPC is a convenient place to test it since we can easily define the
+// callee to throw an exception.)
```

Comment:
A comment here clearly indicating that this is the local stack would be helpful. Just as I briefly made the mistake thinking that the stack would include the remote frames, it would be good to set expectations for folks coming into the code later.